### PR TITLE
Step 2 angular release ready

### PR DESCRIPTION
**Pull request by Tanya0609**
** When PR #770 is merged, then this PR can be merge.**

This PR is to update the GitHub main page, acknowledgement page and the page in the marketplace to include Angular as a frontend framework. Also, this PR move Angular out of preview mode.

**How to test:**
1. (make sure you close your preview mode in vs code extensions)
2. start extension, you should be able to see Angular is one of the frontend frameworks option
3. check all three .MD files has no spelling mistakes, names are sorted in alphabetic order (on first name), links are working

### CODE REVIEW

**Review by piotrmark — DISMISSED** [review]
❤

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ I need a full-stack web app that stores data in a Cosmos database.
 Web Template Studio approaches web app creation using the following four attribute sets:
 
 - **Project type**: First, what type of application are you building? We currently support only one type: _Fullstack Application_.
```
Comment (jasmingoh) [inline README.md]: Should we also update this since we removed Project Type page?

File: src/extension/README.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ Docs to provide useful insights.
 Web Template Studio approaches web app creation using the following four attribute sets:
 
 - **Project type**: First, what type of application are you building? We currently support only one type: _Fullstack Application_.
```
Comment (jasmingoh) [inline src/extension/README.md]: here as well

File: README.md (hunk 1 above)
Comment (Tanya0609) [reply README.md]: good catch! PR updated.

**Comment by Tanya0609** [comment]
Those changes has been logged in #774, #774 also has an additional change. Thus, close this PR

**Pull request closed without merging**